src/WebUI/WorkLoads/DeploymentsTable.tsx

```diff
@@ -255,8 +256,28 @@ export class DeploymentsTable extends BaseComponent<IDeploymentsTableProperties,
         return renderTableCell(rowIndex, columnIndex, tableColumn, itemToRender);
     }
 
-    private static _renderPodsCountCell(rowIndex: number, columnIndex: number, tableColumn: ITableColumn<IDeploymentReplicaSetItem>, deployment: IDeploymentReplicaSetItem): JSX.Element {
-        return renderPodsStatusTableCell(rowIndex, columnIndex, tableColumn, deployment.pods, deployment.statusProps, deployment.podsTooltip);
+    private _renderPodsCountCell = (rowIndex: number, columnIndex: number, tableColumn: ITableColumn<IDeploymentReplicaSetItem>, deployment: IDeploymentReplicaSetItem): JSX.Element => {
+        const replicaSet = this._getSelectedReplicaSet(deployment);
+        const podslist = StoreManager.GetStore<PodsStore>(PodsStore).getState().podsList;
+        const relevantPods = (podslist && podslist.items && podslist.items.filter(p => (replicaSet && Utils.isOwnerMatched(p.metadata, replicaSet.metadata.uid)) || false))
+
+        const _onPodsClicked = (relevantPods && replicaSet) ? (() => {
+            const selectedPod = relevantPods.find(p => PodPhaseToStatus[p.status.phase] === Statuses.Failed) || relevantPods[0];
```

Comment:
in error calculation need to use following function
https://github.com/Microsoft/azpipelines-kubernetesUI/blob/c1561feaa5402887297240d95ec7193db4100089/src/WebUI/Utils.ts#L97

so we know what is the error one